### DISCUSSION TITLE
Unable to convert csvfile into json using nifi

### DISCUSSION THREAD

**Discussion by LavanyaKR3007** [Sunbird-cQube/community#38] [Q&A] [answered]
Hi @krishnakr6 @SreenivasNimmagadda 

We have created a flow file & other processors for converting csv into json . we have created the schema according to the new use case as well.

Json file conversation didn't happened. 

Kindly help us on this.

cc @https://github.com/Ramya-kmurthy

**Comment by SreenivasNimmagadda** [comment, marked as answer]
@LavanyaKR3007,

Your question seems the coding issue. I hope your question is not related to the cQube issue
Please check you NIFI processors, if any configurations were missed.

Thanks.

**Comment by LavanyaKR3007** [comment]
steps followed based on usecase : 1) Adding the Processor for GetFile 2) Add the processor to convert Record 3) Add Log Attribute 4) Add Put File

The Solution 
Added necessary properties & made configurations changes in Nifi for converting CSV into JSON. It was working fine.

Thanks @krishnakr6 for guidance

cc @https://github.com/Ramya-kmurthy
